data_steward/cdr_cleaner/cleaning_rules/deid/conflicting_hpo_state_generalization.py

```diff
@@ -0,0 +1,255 @@
+"""
+Cleaning rule to generalize conflicting HPO states
+"""
+import logging
+
+from google.cloud import bigquery
+
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule, query_spec_list
+from common import JINJA_ENV, OBSERVATION, PIPELINE_TABLES
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+import resources
+
+LOGGER = logging.getLogger(__name__)
+
+JIRA_ISSUE_NUMBERS = ['DC512', 'DC834']
+JIRA_ISSUE_URL = [
+    'https://precisionmedicineinitiative.atlassian.net/browse/DC-512',
+    'https://precisionmedicineinitiative.atlassian.net/browse/DC-834'
+]
+
+MAP_TABLE_NAME = SCHEMA_MAP_TABLE = "person_src_hpos_ext"
```

Comment:
@ratuagga there is some usage inconsistency with this naming convention that I didn't pick up on earlier.  We are creating a lookup table in a sandbox dataset.  Prior history had this table created in the combined dataset to prevent the table from being accidentally published.  That's why the name started with `_mapping`.  Now, we still don't want to publish this lookup table but you are creating it in a sandbox.  We incorrectly assumed the table was named that way because it was drawing data from those specific source tables.  It was actually named that way to prevent it from being published.  So, I would suggest changing this table name to `person_src_hpos_lookup`.